src/main/java/duke/Event.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+
+/**
+ * Adds on with an at field which allows users to set an event timing.
+ *
+ * @author Yuvaraj Kumaresan
+ */
+public class Event extends Task {
+
+    /**
+     * Time at which the event is at.
+     */
+    protected String at;
```

Comment:
Perhaps you could be more specific with the naming of the at variable as it may not be clear to others ?
```suggestion
    protected String eventAt;
```